tests/unit/legal-api-mixin.spec.ts

```diff
@@ -116,7 +115,7 @@ describe('Legal API Mixin', () => {
       .returns(new Promise(resolve => resolve({ data: DIRECTORS })))
 
     // call method
-    const directors = await vm.fetchParties('CP1234567', Roles.DIRECTOR)
+    const directors = await vm.fetchParties('CP1234567', 'Director')
```

Comment:
Tests shouldn't use enums so that they're more fragile (otherwise, an error in the enum would not be caught).